frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -156,7 +156,7 @@ void main(List<String> args) async {
       fail("exception not thrown");
     } catch (e) {
       print('dart catch e: $e');
-      expect(e, isA<FfiException>());
+      expect(e, isA<Exception>());
```

Comment:
Well I hope FfiException still does exist, such that we have less breaking changes

Reply:
I don't know how FfiException would fit in the new PR. Any ideas?

Reply:
Well I mean `FrbException` (given your rename).

Reply:
Every exception is a FrbException. Can I mark this as resolved then?

Reply:
Well I mean write code as `expect(e, isA<FrbException>());` instead of `expect(e, isA<Exception>());`, since we are expecting indeed a FrbException